tests/test-war-smoke/src/test/java/com/google/cloud/runtimes/jetty/test/smoke/ForwardedIntegrationTest.java

```diff
@@ -40,44 +40,6 @@
    * @throws IOException test in error
    */
   @Test
-  @LocalOnly
```

Comment:
Because locally now the gcp module is not enabled, nor is it's http-forwarded dependency.   We could enable it, but then we are just testing the normal jetty http-forwarded module.